src/main/java/com/pretz/dyntreeserver/domain/User.java

```diff
@@ -26,6 +24,11 @@
     public User() {
     }
 
+    public User(UserDTO userDTO) {
```

Comment:
hmm, I'm not sure `User` should depend on `UserDTO`.

Maybe add a method to `UserDTO` called `toModel()`, containing sth like:

```
public User toModel() {
  return new User(this.fieldA, this.fieldB);
}
```
?

just a suggestion. Or even better - don't create dependencies in any of these files and create a external "translator" utility which does DTO <-> Model mapping?

Reply:
I can use normal constructor and pass name and email from DTO as parameters.